### ISSUE TITLE
Add length option to transmission

### ISSUE THREAD

**Issue by brnomendes** [markvader/ha-rpi_rf#10]
The `code_on`/`code_off`, `protocol`, and `pulselength` options are currently supported.

So, add `length` option too.

---

The rpi_rf  0.9.7 already supports the `tx_length` parameter in transmission, the method signature:
```python
def tx_code(self, code, tx_proto=None, tx_pulselength=None, tx_length=None):
```
https://github.com/milaq/rpi-rf/blob/0.9.7/rpi_rf/rpi_rf.py#L91

**Comment by markvader** [comment]
I've added a new branch with length added. https://github.com/markvader/ha-rpi_rf/tree/Length

I'm not able to test it currently so won't merge to master for now.

**Comment by markvader** [comment]
I have merged the branch and released a new version as a beta 2022.4.0-beta.
